lib/src/asset/exceptions.dart

```diff
@@ -11,3 +12,32 @@ class AssetNotFoundException implements Exception {
   @override
   String toString() => 'AssetNotFoundException: $assetId';
```

Comment:
Consider reusing [`Error#safeToString`](https://api.dartlang.org/1.12.0/dart-core/Error/safeToString.html) in exception `toString` methods.